### PR TITLE
Mention NixOS packaging in readme

### PR DESCRIPTION
**Pull request by expipiplus1**
Up to you to decide if this is readme-worthy, I won't mind either way!

https://github.com/NixOS/nixpkgs/pull/121130

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,8 +28,9 @@ directly to the ECP5 pins).
 ## Pre-built Binaries
 
 Pre-built binaries are available for Windows and Linux on the [Releases] page.
-You must have [libusb] installed or available on your system, and you must
-have permissions or drivers set up to access your CMSIS-DAP probe.
+ECPDAP is also packaged for NixOS under the `ecpdap` attribute. You must have 
+[libusb] installed or available on your system, and you must have permissions 
+or drivers set up to access your CMSIS-DAP probe.
```
Comment (adamgreig) [inline README.md:33]:
```suggestion
You must have [libusb] installed or available on your system, and you must
have permissions or drivers set up to access your CMSIS-DAP probe.

ECPDAP is also packaged for NixOS under the `ecpdap` attribute.
```

**Comment by adamgreig** [comment]
Cool, thanks! Happy to include this in the readme, I've suggested a slightly different wording to make it clearer the libusb requirement isn't related to NixOS.

I'll try and remember to check the Cargo.lock is up to date before the next release... I suppose it's no help doing a new release just for that now?

**Comment by expipiplus1** [comment]
Thanks for the clarifying change!

Actually a new release now would be good actually, I can drop the lockfile patch (so that automatic updates happen cleanly) and can set it up so that the udev rules are incorporated into NixOS without having to do it manually.

**Comment by expipiplus1** [comment]
Thanks!

**Comment by expipiplus1** [comment]
https://github.com/NixOS/nixpkgs/pull/121275

**Pull request merged**